src/Job/Job.ts

```diff
@@ -13,30 +13,43 @@ export interface JobEnv {
  */
 export interface Job {
 	/**
-	 * Get this job's name.
+	 * Get this job's target.
 	 *
-	 * This method may be removed in the future because Jobs may no longer have names.
+	 * The target is the resulting file.
 	 *
-	 * @experimental
-	 * @returns This job's name.
+	 * @returns This target.
 	 */
-	getName(): string
+	getTarget(): string
```

Comment:
Maybe I could set the original method `getName()` as `@deprecated` and add `getTarget()` to return the same thing. I want to add the `getTarget()` somewhere because someone running a Job and trying to find its target shouldn't have to know that its name is actually its target.

Thoughts?

Reply:
Sure that's a good way to do it. Backwards compatible while still encouraging the new method.